app/scripts/fornac.js

```diff
@@ -1654,6 +1657,19 @@ export function FornaContainer(element, passedOptions) {
         self.updateStyle();
     };
 
+    self.getGetTitleText = function (d) {
```

Comment:
Can we change this to just `getTitleText` instead?

Reply:
Yes, sure. Sorry, that's my bad.